code/game/machinery/computer/camera_advanced.dm

```diff
@@ -75,7 +75,7 @@
 		for(var/obj/machinery/camera/C in GLOB.cameranet.cameras)
 			if(!C.can_use())
 				continue
-			if(C.network&networks)
+			if(length(C.network & networks))
```

Comment:
So what happens with this now.

Does it start on Z2 or what?

Reply:
Oh yeah, probably should've explained this, huh?

Short explanation:
This is the code which decides where the camera 'Eye' is spawned initially, with this change it only selects cameras who's `network` contains one of the same things from the console's `networks`. In this case, `list("SS13", "Abductor")`.
This bug was most likely introduced by map standardisation/rotation, since they changed around the load order.

<details>
<summary><b>Long explanation:</b></summary>

In this case the camera it was always connecting to had the networks `list("ERT", "CentComm")`. Obviously neither of those match what the camera console has.
The reason for this is that the `&` operator was being used incorrectly, specifically this part:
> [If A and B are lists, the result is a list that contains only items that were in both lists, in the order of list A.](http://www.byond.com/docs/ref/#/operator/&)

Even though nothing inside the list matched, it was still returning an empty list, and the `if()` was taking that as confirmation.
```dm
// Before
var/list/L1 = list("ERT", "CentComm")
var/list/L2 = list("SS13", "Abductor")
var/list/L3 = (L1 & L2) // << Produces an empty `list()`.
if(L3) // The list exists.
	camera_location = get_turf(C)
```
```dm
// After
var/list/L1 = list("ERT", "CentComm")
var/list/L2 = list("SS13", "Abductor")
var/list/L3 = (L1 & L2) // << Produces an empty `list().
if(length(L3)) // Length of 0.
	camera_location = get_turf(C)
```

</details>

Reply:
Ah listing. I thought it was bitflags at first since `&`. Makes sense.